### PR TITLE
Add advanced usage doc for programatically comparing models

### PR DESCRIPTION
**Pull request by consoull**
### Linked issue(s)
Fixes KOL-6428

### What change does this PR introduce and why?
Adds advanced usage doc for programatically comparing models via the sdk `download_quality_standard_result`

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [x] Relevant docs have been added / updated

### CODE REVIEW

File: docs/dataset/advanced-usage/quality-standard-results.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+The return value is a multi-index DataFrame with indices `(stratificaiton, test_case)` and columns `(model, eval_config,
+metric_group, metric)`.
+
+<figure markdown>
+![Quality Standard Result](../../assets/images/‎quality_standard_diagram_light.jpeg#only-light)
+![Quality Standard Result](../../assets/images/‎quality_standard_diagram_dark.jpeg#only-dark)
+<figcaption>Quality Standard Result</figcaption>
+</figure>
+
+## Examples
```
Comment (consoull) [inline docs/dataset/advanced-usage/quality-standard-results.md]: Working on another example for custom reporting eg. automate a weekly email with bottom 10 performing test cases to boss

Comment (consoull) [reply docs/dataset/advanced-usage/quality-standard-results.md]: Going to scratch this example for now

File: docs/dataset/advanced-usage/quality-standard-results.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+icon: kolena/take-action-16
+
+---
+
+# :kolena-take-action-20: Programatically Compare Models
+
+## Download Quality Standard Result
+
```
Comment (ly-kolena) [inline docs/dataset/advanced-usage/quality-standard-results.md:13]: Do we want to add a note that this is experimental feature? 
```
!!! example "Experimental Feature"
```

File: docs/dataset/advanced-usage/quality-standard-results.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+The **Quality Standard** should contain the key performance metrics a team uses to evaluate a model's performance on a
+dataset.
+
+The SDK provides [a
+function](../../reference/experimental/index.md#kolena._experimental.quality_standard.download_quality_standard_result)
```
Comment (ly-kolena) [inline docs/dataset/advanced-usage/quality-standard-results.md]: Nit: would this be clearer?
```
provides a function [download_quality_standard_results](...)
```

**Review by ly-kolena — APPROVED** [review]

File: docs/dataset/advanced-usage/quality-standard-results.md (hunk 2 above)
Comment (consoull) [reply docs/dataset/advanced-usage/quality-standard-results.md:13]: Good call

**Pull request merged**